### DISCUSSION TITLE
Is there a way to set max CPU temperature & bottleneck settings?

### DISCUSSION THREAD

**Discussion by Animeshz** [AdnanHodzic/auto-cpufreq#267] [Q&A]
I want to hit max 80 degrees Celcius and then bottleneck until it reaches like 75, is there a way to do that?

**Comment by Animeshz** [comment]
Also can we use tools like ryzen-controller / ryzen-adj in combination with this? It seems that it does have such settings in it (it doesn't let the temp fall but it also doesn't let the temps cross the limit, its like stable 80 degrees at maximum load if set). I'm not sure if they can conflict with each other.

**Comment by AdnanHodzic** [comment]
Currently there's no way to limit, but you could potentially disable turbo as [part of config file](https://github.com/AdnanHodzic/auto-cpufreq#configuring-auto-cpufreq) and see if that makes any difference?
